Main.java

```diff
@@ -10,6 +12,21 @@
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+    System.out.println ("How many chromebook chargers would you like");
+    int Chromebook_chargers =input.nextInt();
+    double Chromebook_chargerstotal  =Chromebook_chargers  * 34.99;
+    System.out.println("How many motherboards would you like");
+    int Motherboard =input.nextInt();
+    double Motherboardtotal =Motherboard * 127.50;
+    System.out.println("How many mice would you like");
+    int Mice =input.nextInt();
+    int Micetotal =Mice*18;
+    double Subtotal = Chromebook_chargerstotal+Motherboardtotal+Micetotal;
+    System.out.println("Your Subtotal is " + Subtotal + "." );
+double taxes = Subtotal*0.13;
+System.out.println("Tax is " + taxes + ".");
+    double total = Subtotal+taxes;
+    System.out.println("Your total is " + total + ".");

```

Comment:
While the math is good, you are missing comments in your code. Also, remember that variables should always begin with a lowercase letter and if it is more than one word, it is camelCased. Finally, make sure your output matches as closely as possible to the sample output given.